src/js/education-wizard.js

```diff
@@ -0,0 +1,61 @@
+  function toggleClass(element, className) {
```

Comment:
Maybe we should put this file in `src/js/edu-benefits`? We don't have a folder for js for content pages aside from `legacy`, which seems misnamed.